Cargo.toml

```diff
@@ -62,80 +62,67 @@ alloy-rpc-types-engine = { version = "0.9.2", default-features = false }
 alloy-network-primitives = { version = "0.9.2", default-features = false }
 
 # Alloy RLP
-alloy-rlp = { version = "0.3", default-features = false }
+alloy-rlp = { version = "0.3.10", default-features = false }
 
 # Alloy Core
-alloy-sol-types = { version = "0.8.12", default-features = false }
-alloy-primitives = { version = "0.8.12", default-features = false }
+alloy-sol-types = { version = "0.8.18", default-features = false }
+alloy-primitives = { version = "0.8.18", default-features = false }
 
 # Revm
-revm = "19.0.0"
+revm = "19.2.0"
 
 # Serde
-serde_repr = "0.1"
-serde = { version = "1.0", default-features = false, features = [
+serde_repr = "0.1.19"
+serde = { version = "1.0.217", default-features = false, features = [
     "derive",
     "alloc",
 ] }
-serde_with = "3.11"
-serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
+serde_with = "3.12.0"
+serde_json = { version = "1.0.135", default-features = false, features = ["alloc"] }
 
 # Encoding
 snap = "1.1.1"
 bincode = "1.3.3"
-ethereum_ssz = "0.8"
+ethereum_ssz = "0.8.2"
 
 # Compression
 miniz_oxide = "0.8.2"
 alloc-no-stdlib = "2.0.4"
 brotli = { version = "7.0.0", default-features = false }
 
 # rpc
-jsonrpsee = { version = "0.24", features = [
+jsonrpsee = { version = "0.24.7", features = [
     "jsonrpsee-core",
     "client-core",
     "server-core",
     "macros",
 ] }
-jsonrpsee-core = "0.24"
-jsonrpsee-types = "0.24"
+jsonrpsee-core = "0.24.7"
+jsonrpsee-types = "0.24.7"
 
 # misc
-cfg-if = "1"
+cfg-if = "1.0.0"
 async-trait = "0.1.85"
 unsigned-varint = "0.8.0"
 spin = { version = "0.9.8", features = ["mutex"] }
 lazy_static = { version = "1.5.0", default-features = false }
-derive_more = { version = "1.0", default-features = false }
-thiserror = { version = "2.0", default-features = false }
-similar-asserts = "1.6"
+derive_more = { version = "1.0.0", default-features = false }
+thiserror = { version = "2.0.9", default-features = false }
+similar-asserts = "1.6.0"
 
 # tracing
 tracing-subscriber = "0.3.19"
 tracing = { version = "0.1.41", default-features = false }
 
 ## misc-testing
-arbitrary = { version = "1.4", features = ["derive"] }
-arbtest = "0.3"
-rand = "0.8"
-proptest = "1.6"
-proptest-derive = "0.5"
-tokio = "1"
+arbitrary = { version = "1.4.1", features = ["derive"] }
+arbtest = "0.3.2"
+rand = "0.8.5"
+proptest = "1.6.0"
+proptest-derive = "0.5.``"
```

Comment:
this looks wrong.. what do the back ticks do?

Reply:
Oh jeez... how is ci passing